assets/sass/base/_defaults.scss

```diff
@@ -146,6 +133,19 @@
 		font-size: 1rem;
 	}
 
+	p.googlesitekit-overline {
+		color: $c-primary;
+		font-size: 0.75rem;
+		font-weight: $fw-primary-medium;
+		letter-spacing: 0.16em;
+		line-height: 1.16;
+		text-transform: uppercase;
+
+		&--collapse {
+			margin: 0;
+		}
```

Comment:
This variant doesn't seem to be used at all so let's remove it.

Reply:
👍